.github/workflows/php.yml

```diff
@@ -0,0 +1,71 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.2'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.4'
+            symfony-version: '^4'
+            stability: ''
```

Comment:
imho we don't need quite that many different builds. i think we want:

* every php version we support without symfony-version
* every symfony version we support with a suitable php version (aka symfony 4 with php 7.2 and symfony 5 with 7.3, 7.4 and 8.0)
* a lowest version build with php 7.2
* a build with stability dev and php 8.0

the goal is to cover all reasonable cases in as few builds as possible.

Reply:
To reduce the number of builds further, feel free to remove PHP 7.2, which is [EOL anyway](https://www.php.net/supported-versions.php).

Reply:
if we drop PHP 7.2 we need to also adjust composer.json to not allow installing the bundle on that php version.

Reply:
Thanks you :) It's done ;)